forms.rst

```diff
@@ -449,47 +424,6 @@ the date as a string in the box)::
 .. image:: /_images/form/simple-form-2.png
     :align: center
 
-Each field type has a number of different options that can be passed to it.
-Many of these are specific to the field type and details can be found in
-the documentation for each type.
-
-.. sidebar:: The ``required`` Option
```

Comment:
What is the reason to remove this section?

Reply:
Well, doesn't contain any relevant information. The documentation is linked anyway.